src/core/qgscolorutils.h

```diff
@@ -94,6 +95,23 @@ class CORE_EXPORT QgsColorUtils
      */
     static QColor colorFromString( const QString &string );
 
+    /**
+     * Load \a iccProfileFilePath and returns associated color space.
+     * If an error occurred, an invalid color space is returned and \a errorMsg is updated with error
+     * message
```

Comment:
\since QGIS 3.40

Also add \param and \returns, so that sipify work correctly with the SIP_OUT argument